primer/brand
# `Label` accessibility fixes

## Summary


Applies fixes stemming from the [review of `<Label>`](https://github.com/github/primer/issues/1971).

## List of notable changes:



- Changes [colors used for gradients](https://primer.style/brand/primitives/color) `Green-blue` and `Red-orange` to use `x.5` instead of `x.4` for improved color contrast.
- Adds outline for high contrast modes.

## What should reviewers focus on?

- Color contrast for `Green-blue` and `Red-orange`.
- See if high contrast mode styles are applied.
(This can be tested within Chrome without Windows through [Chrome Devtools via the "Rendering" tab](https://developer.chrome.com/docs/devtools/rendering/emulate-css/#emulate-css-media-feature-forced-colors). Within the "**Emulate CSS media feature forced-colors**" section, select the "**forced-colors: active**" option in the select menu).

## Steps to test:



1. Open up `Label` color example in PR provided environment.
1. Observe `Green-blue` and `Red-orange` changes
1. Observe `Label` outline when in high contrast mode (see "What should reviewers focus on?" section on how to test in HCM).

## Supporting resources (related issues, external links, etc):

- [`Label` review (internal)](https://github.com/github/primer/issues/1971)
- [Windows High Contrast Mode & development](https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

| Before | After |
|--------|--------|
| <img width="96" alt="A label with a border surrounding the text 'label', the label has a gradient background color spanning from light green to dark blue" src="https://github.com/primer/brand/assets/26746305/35744f0b-1d42-410d-bf0b-4ae6083bc2d7"> <br /> Green color with `.4` variant. | <img width="96" alt="A label with a border surrounding the text 'label', the label has a gradient background color spanning from a dark green to dark blue" src="https://github.com/primer/brand/assets/26746305/a7966c6b-c469-494c-b9dc-6edd8fd614eb"> <br /> Green color changes to slightly darker variant, from `.4` to `.5`. |
| <img width="96" alt="Label component in an emulated high contrast mode theme, the label has no outline present" src="https://github.com/primer/brand/assets/26746305/3bad97f6-2d81-446a-9379-f374f8966a2e"> <br /> `Label` in high contrast mode. | <img width="96" alt="Label component in an emulated high contrast mode theme with a white outline" src="https://github.com/primer/brand/assets/26746305/5e4e3a5d-39ad-41a8-9204-0a272a001af3"> <br /> `Label` in high contrast mode with an outline present. |
